test/parallel/test-http-dont-override-options.js

```diff
@@ -0,0 +1,43 @@
+const common = require('../common');
```

Comment:
It didn't originally error for me, but I also thought so too so I added `'use strict';` in the new commit.